gatsby/src/pages/borderline-substances/{BnfBorderlineSubstancesTaxonomy.slug}.module.scss

```diff
@@ -0,0 +1,44 @@
+@use '@nice-digital/nds-core/scss/core';
+
+.nav {
+  background: core.$nds-colour-nice-white;
+  border: 1px solid core.$nds-colour-nice-grey-2;
+  margin-bottom: core.rem(core.$nds-spacing-large);
+  padding: core.rem(core.$nds-spacing-medium);
+
+  &.navReadableMaxWidth {
+    max-width: core.$nds-readable-width;
+  }
+
+  @media print {
+    display: none;
+  }
+}
+
+.linkList {
```

Comment:
This linkList looks quite like the `ColumnList` from the design system. Can we use that instead?